rbac/management/principal/view.py

```diff
@@ -183,12 +184,15 @@ def get(self, request):
                 count = len(data)
             else:
                 count = None
+            last_link_offset = int(count) - int(limit) if (int(count) - int(limit)) >= 0 else 0
```

Comment:
Nit: we can put int(count) - int(limit) into a variable so we don't have to calculate it twice